bower.json.erb

```diff
@@ -2,7 +2,7 @@
   "name": "rad-consumer",
   "dependencies": {
     "dough": "<%= gem_path('dough-ruby') %>",
-    "yeast": "1.14.0",
+    "yeast": "git@github.com:moneyadviceservice/yeast.git#95a3953e49a92f380086a9c12782d7e92a0168dc", 
```

Comment:
I'm guessing you'll be using the same changes in `yeast` that you did for `frontend`?

Reply:
Yes, that will be replaced by a new version for deployment.